--- archlinux-linux/init/Kconfig.orig	2019-03-12 10:11:33.248854943 +0100
+++ archlinux-linux/init/Kconfig	2019-03-12 10:11:54.145524086 +0100
@@ -922,7 +922,7 @@
 
 config CGROUP_CPUACCT
 	bool "Simple CPU accounting controller"
-	depends on !SCHED_BMQ
+	depends on SCHED_BMQ
 	help
 	  Provides a simple controller for monitoring the
 	  total CPU consumed by the tasks in a cgroup.
--- archlinux-linux/kernel/sched/Makefile.orig	2019-03-12 13:50:40.225012624 +0100
+++ archlinux-linux/kernel/sched/Makefile	2019-03-12 13:51:12.565130138 +0100
@@ -24,8 +24,8 @@
 obj-$(CONFIG_SMP) += cpudeadline.o topology.o stop_task.o
 obj-$(CONFIG_SCHED_AUTOGROUP) += autogroup.o
 obj-$(CONFIG_SCHED_DEBUG) += debug.o
-obj-$(CONFIG_CGROUP_CPUACCT) += cpuacct.o
 endif
+obj-$(CONFIG_CGROUP_CPUACCT) += cpuacct.o
 obj-y += loadavg.o clock.o cputime.o
 obj-y += idle.o
 obj-y += wait.o wait_bit.o swait.o completion.o
